### PR TITLE
dec dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-december-2021.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+Title: iRODS Development Update: December 2021
+Date: 2021-12-21 12:00
+Author: Kory Draughn
+Slug: irods-development-update-december-2021
+Status: published
+
+
+iRODS v4.2.11 has been released! You can read all about it [here](irods-4-2-11-is-released.md).
```
Comment (trel) [inline content/posts/irods-development-update-december-2021.md:8]: this renders to a 404...  try...

```
[here]({filename}/posts/irods-4-2-11-is-released.md)
```

File: content/posts/irods-development-update-december-2021.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+For this month's development update, we've started shifting focus to iRODS 4.3. Work on the 4.2.x branch is coming to an end. The goal is to have 4.3 tested and released by next year's UGM. With that in mind, let's start by talking about what has changed for the 4.3 branch ...
+
+First off, [irods/externals](https://github.com/irods/externals) has been updated to require Python 3. Not only that, but it has gained support for Ubuntu 20, Debian 11, AlmaLinux 8, and Rocky Linux 8. Support for Ubuntu 14, Debian 9, and Debian 10 have been dropped.
+
+The Python iRODS Client has received some improvements as well. Just like irule, it now supports targeting a specific rule engine plugin instance when executing rules. It also supports switchable XML parsers. This change allows the PRC to handle XML encoding issues with single quotes and apostrophes seen in earlier versions of iRODS.
```
Comment (trel) [inline content/posts/irods-development-update-december-2021.md:14]: can use backticks.... like
```
`irule`
```

**Review by trel — COMMENTED** [review]
looks good - two tweaks, and we'll merge in the morning.

**Comment by korydraughn** [comment]
updated!

**Comment by trel** [comment]
confirmed.

**Pull request merged**